webpack/scenes/ContentViews/Details/Repositories/__tests__/contentViewAddRemove.test.js

```diff
@@ -92,16 +92,19 @@ test('Can add repositories', async (done) => {
   expect(getByLabelText('add_repositories')).toHaveAttribute('aria-disabled', 'true');
   getByLabelText('Select all rows').click();
   getByLabelText('add_repositories').click();
+  await patientlyWaitFor(() => expect(getByText(firstRepo.name)).toBeInTheDocument());
+
   assertNockRequest(repoAddscope);
 
   assertNockRequest(cvDetailScope);
   assertNockRequest(autocompleteScope);
   assertNockRequest(scope, done);
+  act(done);
```

Comment:
I still don't understand this.. 😕

Reply:
We add this to tell the test to ignore further actions on the render, so in this case there is still another api call after the "scope, done". 

But we don't care about that, so we specifically tell the test that we are done all actions: act(done). 
In this case, without the act(done) call, the test would wait forever for a corresponding api call to the follow up one that we don't care about.

Reply:
This is one of those cases where a change we have made, has a few chained api calls, sometimes we want to wait and confirm them, I didn't in this case, so I just told the test to STAHP.